desc/plotting.py

```diff
@@ -2082,6 +2093,28 @@ def plot_boundaries(
         fig, ax = plot_boundaries((eq1, eq2, eq3))
 
     """
+    NFPs = np.array([thing.NFP for thing in eqs])
+    Ns = np.array([thing.N for thing in eqs])
+    if not np.allclose(NFPs, NFPs[0]) and np.any(Ns == 0):
+        # if all NFPs are not equal, maybe there are some axisymmetric
+        # objects. We can try to change those to match the NFP of the first
+        # of the nonaxisymmetric objects
+        eqs = [thing.copy() for thing in eqs]  # make copy so we dont modify originals
+        NFP_nonax = int(NFPs[NFPs > 1][0])
+        [
+            thing.change_resolution(NFP=NFP_nonax if thing.N == 0 else thing.NFP)
+            for thing in eqs
+        ]
```

Comment:
can simplify all this stuff by just requiring that `phi` be given explicitly whenever `NFP` differs.